pkg/controller/redirect/helpers_test.go

```diff
@@ -319,18 +319,26 @@ func TestDecideRedirect(t *testing.T) {
 			appStoreData: &appLinkNeither,
 			expected:     "",
 		},
+		{
+			name:         "android_onboarding",
+			url:          "https://moosylvania.gov/",
+			enxEnabled:   false,
+			userAgent:    userAgentAndroid,
+			appStoreData: &appLinkNeither,
+			expected:     "market://search?q=exposure%20notifications",
+		},
 
 		// iOS
 		{
 			name:         "ios_both",
-			url:          "https://ios.example.com/store/moosylvania",
+			url:          "https://ios.example.com/store/moosylvania?c=123456",
```

Comment:
do we want the code in that example?

Reply:
The previous implementation did not redirect to the appstore with the code. We could add it, but that would be a behavior change (and it would be sharing the code with external servers - they would show up in Apple's App Store request logs). WDYT?

Reply:
@mikehelmick ^

Reply:
right - that's what I'm worried about - I don't think it's useful to send the code to the store and have it in the logs

Reply:
I see the confusion. This is the _incoming_ url. The outgoing url is below, sans code.